_includes/pbc/all/install-features/202304.0/install-warehouse-picking-feature.md

```diff
@@ -0,0 +1,534 @@
+
+class PickingListDependencyProvider extends SprykerPickingListDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\PickingListExtension\Dependency\Plugin\PickingListGeneratorStrategyPluginInterface>
+     */
+    protected function getPickingListGeneratorStrategyPlugins(): array
+    {
+        return [
+            new MultiShipmentPickingListGeneratorStrategyPlugin(),
+        ];
+    }
+}
+```
+
+3. To enable the Backend API, register the plugins:
+
+| PLUGIN                                | SPECIFICATION                                | PREREQUISITES | NAMESPACE                                                            |
+|---------------------------------------|----------------------------------------------|---------------|----------------------------------------------------------------------|
+| PickingListsBackendResourcePlugin     | Registers the `picking-lists` resource.      |               | Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication |
+| PickingListItemsBackendResourcePlugin | Registers the `picking-list-items` resource. |               | Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication |
+
+**src/Pyz/Glue/GlueBackendApiApplication/GlueBackendApiApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueBackendApiApplication;
+
+use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
+use Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication\PickingListsBackendResourcePlugin;
+use Spryker\Glue\PickingListsBackendApi\Plugin\GlueBackendApiApplication\PickingListItemsBackendResourcePlugin;
+
+class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new PickingListsBackendResourcePlugin(),
+            new PickingListItemsBackendResourcePlugin()
+        ];
+    }
+}
+
+```
+
+{% info_block warningBox "Verification" %}
+
+As a prerequisite, pass the following steps:
+
+* Attach a Back Office user to any warehouse you have in the system. For this, use the [Install the Warehouse User Management feature](/docs/scos/dev/feature-integration-guides/{{site.version}}/install-the-warehouse-user-management-feature.html) guide.
+* Place an order in the system so that the product is in the warehouse where you added the user in the previous step.
+* Obtain the access token of the warehouse user.
+* Use the warehouse user access token as the request header: `Authorization: Bearer {{your access token}}`.
+
+Make sure that you can send the following requests:
+
+To get a collection of available picking lists for a warehouse user:
```

Comment:
incomplete sentence, add the second part to make it complete—for example:
```suggestion
To get a collection of available picking lists for a warehouse user, send the request:
```

Adjust all the sentences below that has the same mistake.